apps/docs/content/components/FAQ/index.mdx

```diff
@@ -51,14 +51,14 @@ When using long lists of questions and answers, consider using subheadings to gr
 <DoDontContainer stacked>
   <Do>
     <img
-      src="https://github.com/primer/brand/assets/912236/e38b0b06-24cd-4d45-8939-1ad31bb62e58"
+      src="https://github.com/primer/brand/assets/912236/028b0aaf-148e-4a7b-b454-c733d1460e49"
```

Comment:
Should this one show the [newer FAQ groups pattern](https://primer-e6cc30237b-26139705.drafts.github.io/components/FAQ/react#grouped-faqs)?

Reply:
Good catch, updated content and images with FAQ groups 👍